package.json

```diff
@@ -9,13 +9,17 @@
     "packages/*"
   ],
   "scripts": {
-    "postinstall": "yarn bootstrap",
+    "postinstall": "yarn bootstrap && patch-package",
```

Comment:
😄  Just did some reading on patches, as I'm new to them. Interesting tooling. Wish I know about this before.

The one `.patch` file I saw accompanying this pull request is for `@vue/test-utils` right?  If I understood correctly, when the patch is finally merged to core, we can update the version of `@vue/test-utils` and remove the patch-package command from the `postinstall` script, right?

Reply:
Correct :-)

We use this sparingly while we wait for upstream merges.